docs/guides/cloud/introduction.mdx

```diff
@@ -4,153 +4,139 @@ sidebar_position: 10
+[command logs, screenshots, video replays, stack traces, and CI logs](/guides/cloud/runs#Test-detail-sidebar).
+Quickly identifying a test failure in CI is just a click away.
 
-## Benefits
+<DocsVideo src="https://vimeo.com/806005654" />
 
-### Analyze and diagnose
+### Analyze and diagnose test health
 
-Store the full history of your test results, with
-[video clips, screenshots, and full stack traces](/guides/cloud/runs#Test-failures).
-Quickly see the current state of your app on the
-[Latest Runs](/guides/cloud/runs) page, identify problematic trends with rich
-[Analytics](/guides/cloud/analytics), and diagnose unreliable tests with
-[Flaky Test Management](/guides/cloud/flaky-test-management). Associate related
-tests with [grouping](/guides/guides/parallelization#Grouping-by-browser) to see
-results broken down by browser and OS.
+View each [run's overview](/guides/cloud/runs#Overview-tab) to see and compare
+past runs and analyze trends over time. You can quickly analyze changes in your
+setup that might introduce problematic trends as well as identify unreliable
+tests with [Flaky Test Management](/guides/cloud/flaky-test-management).
 
-<DocsImage
-  src="/img/dashboard/dashboard-runs-list.png"
-  alt="Cloud Screenshot"
-/>
+<DocsVideo src="https://vimeo.com/806900039" />
 
-For users of the open-source [Cypress app](/guides/core-concepts/cypress-app),
-we've integrated test run information from Cypress Cloud so developers can see
-the latest results across the team, and identify areas of concern.
+The [analytics](/guides/cloud/analytics) page shows test metrics over time,
+helping determine the overall health of your test suite. Quickly see which tests
+are problematic, getting slower, or growing over time.
 
 <DocsImage
-  src="/img/dashboard/v10/runs-list-in-cypress-app.png"
-  alt="Runs List"
+  src="/img/guides/cloud/analytics/analytics-runstatus.jpg"
+  alt="Analytics run status"
 />
 
-### Run tests in parallel, in priority order, or not at all
+### Reduce test time & CI costs
 
-With our [Smart Orchestration](/guides/cloud/smart-orchestration) features, you
-can run tests across a swarm of machines simultaneously while Cypress Cloud
-coordinates runners and
-[balances test loads](/guides/guides/parallelization#Balance-strategy) - no
-setup required! You can prioritize recently failed specs with
-[Spec Prioritization](/guides/cloud/smart-orchestration#Spec-Prioritization) to
-surface problems earlier, and cancel whole test runs on failure with
-[Auto Cancellation](/guides/cloud/smart-orchestration#Auto-Cancellation) to save
-on resource usage. You can also
-[cancel in-progress runs](/guides/cloud/runs#Cancel-run) manually from Cypress
-Cloud if you need to.
+[Smart Orchestration](/guides/cloud/smart-orchestration) enables you to run
+tests across multiple machines simultaneously in your CI environment while
+Cypress Cloud coordinates runners and
+[balances test loads](/guides/guides/parallelization#Balance-strategy)!
```

Comment:
This note about not having to set it up was part of the original copy and feels like a big detail. 

```suggestion
[balances test loads](/guides/guides/parallelization#Balance-strategy) - no setup required!
```

Reply:
I took it out because I didn't think it was true, there is setup required to do parallelization/load balancing.

Reply:
I always took that to be mean the load balancing portion is handled under the hood by Cypress, like you don't need to orchestrate that. Are you talking about the parallelization set up?

Reply:
good point, i added it back in